### ISSUE TITLE
Open analysis data | Database

### ISSUE THREAD

**Issue by r3sus** [REDasmOrg/REDasm#50]
Hi. 
Please, explain, how do I open the analysis data (.rds). Opening it directly = analyzing, exe = repeat. 
Also, please, clarify what is Database for. I thought analysis data can be open from here, but I found it empty and searching here led to crash. 
[gif](https://user-images.githubusercontent.com/49620111/134222290-eb924a75-6f2d-4de8-a589-f2c41c2f952e.gif)
Win 7. last dev build
Thanks.

**Comment by Dax89** [comment]
The database parts (file format and API) are not standardized yet, it's one of the reasons why REDasm 3.0 is not released as a stable.

Currently the data is simple json, later is converted to msgpack and compressed with ZLib, REDasm is able to load plain JSON too.
For example, these databases are working on REDasm: https://github.com/REDasmOrg/REDasm-Database/tree/master/loaders


**Context Help**
Sure, I can add tooltips.

**Interface Settings**
I can check that, but it's on low priority atm.

**HotKeys**
- Do you mean pg up/down in the plain listing view?
- The CTRL+C issue is in the XRefs dialog or in plain listing view (it's a focus issue?) ?

**Memory Leak**
Yes currently it's not possible to stop an analysis gracefully, it will be fixed during the beta cycle.


Thanks for the feedback btw!

**Comment by r3sus** [comment]
Yes, pg up/dn in listing: [gif](https://user-images.githubusercontent.com/49620111/134332323-f1339958-9b91-4c28-b577-391bca5ba6b2.gif).
The ^C hotkey works fine today. Earlier I had the issue in listing: even after highlighting text, ^C didn't copy it.
So, for now, after closing file have to be reanalyzed?
Thank you.

**Comment by Dax89** [comment]
> So, analysis data can be only saved for now, load is not implemented yet, and file have to be reanalyzed every time after closing?

Yes, exactly, files have to be reanalyzed, I see if I can implement it with BETA 6 or 7

> Also, fancy copying of listing would be nice

Nice! This can be done.
